components/dashboard/src/admin/Settings.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useContext } from "react";
+import { AdminContext } from "../admin-context";
+import CheckBox from "../components/CheckBox";
+import { PageWithSubMenu } from "../components/PageWithSubMenu";
+import { getGitpodService } from "../service/service";
+import { adminMenu } from "./admin-menu";
+
+export default function Settings() {
+    const { sendTelemetry, setSendTelemetry } = useContext(AdminContext);
+
+    const actuallySetTelemetryPrefs = async (value: boolean) => {
+        await getGitpodService().server.adminSettingsSetTelemetry(value);
+        setSendTelemetry(value);
+    }
+
+    return (
+        <div>
+            <PageWithSubMenu subMenu={adminMenu} title="Settings" subtitle="Configure settings for your Gitpod cluster.">
+                <CheckBox
+                    title="Send anonymous service ping to Gitpod"
```

Comment:
1st questions that pops up here is: "What exactly is sent?" Either explained here, or with a ref to docs.

Reply:
Good question. We had a @gitpod-io/engineering-self-hosted [discussion on this](https://gitpod.slack.com/archives/C01KLC56NP7/p1642696983023300) and agreed that wording was enough to be going on with.

Is there a way of putting a link to the [privacy policy](https://www.gitpod.io/privacy) in the description?